src/tabnineChatWidget/handlers/navigateToLocation.ts

```diff
@@ -0,0 +1,38 @@
+import * as vscode from "vscode";
+
+export interface NavigateToLocationPayload {
+  path: string;
+  range: {
+    startLine: number;
+    endLine: number;
+  };
+}
+export function navigateToLocation(payload: NavigateToLocationPayload): void {
+  void doNavigate(payload);
+}
+
+async function doNavigate({
+  path,
+  range,
+}: NavigateToLocationPayload): Promise<void> {
+  const uri = vscode.Uri.file(path);
+  const document = await vscode.workspace.openTextDocument(uri);
+  await vscode.window.showTextDocument(document);
+  const editor = vscode.window.visibleTextEditors.find(
+    (visibleEditor) => visibleEditor.document === document
+  );
+  if (!editor) return;
+
+  const endLineLength = document.lineAt(range.endLine).text.length;
```

Comment:
it is minor, but if you need the whole last line, then why not just add 1 to the endLine and  start from zero position at the (endLine+ 1) line? 
then calculate the range as following:
```
const navigationRange = new vscode.Range(
    range.startLine,
    0,
    range.endLine+ 1,
    0
  );

Reply:
in IJ i implemented it this way, and would like to do the same here for consistency.

Reply:
nvm, you've changed the navigateToLocation